docs/root/configuration/http/http_conn_man/response_code_details.rst

```diff
@@ -38,7 +38,7 @@ Below are the list of reasons the HttpConnectionManager or Router filter may sen
    missing_path_rejected, The request was rejected due to a missing Path or :path header field.
    no_healthy_upstream, The request was rejected by the router filter because there was no healthy upstream found.
    overload, The request was rejected due to the Overload Manager reaching configured resource limits.
-   original_ip_detection_failed, The request was rejected because the original IP couldn't be detected.
+   rejecting_because_detection_failed, The request was rejected because the original IP couldn't be detected.
```

Comment:
```suggestion
   ``rejecting_because_detection_failed``, The request was rejected because the original IP couldn't be detected.
```

Reply:
hmm - i see its in a table - feel free to ignore my suggestion - we probably just want to update them all at the same time

Reply:
I think that the formatting should be changed in a different pull request.

Reply:
yep, cool, wfm, thanks